pkgs/applications/window-managers/velox/default.nix

```diff
@@ -1,25 +1,53 @@
-{ lib, stdenv, fetchurl, fetchFromGitHub, pkgconfig
-, swc, libxkbcommon
-, wld, wayland, pixman, fontconfig
+{ lib, stdenv, fetchFromGitHub, pkgconfig, makeWrapper, newScope
+, libxkbcommon
+, wayland, pixman, fontconfig
+, stConf ? null, stPatches ? []
 }:
 
-stdenv.mkDerivation rec {
+let
+  callPackage = newScope self;
+  self = {
+    swc = callPackage ./swc.nix {};
+    wld = callPackage ./wld.nix {};
+    dmenu-wayland = callPackage ./dmenu.nix {};
+    st-wayland = callPackage ./st.nix {
+      conf = stConf;
+      patches = stPatches;
+    };
```

Comment:
why not add these to all-packages.nix?

Reply:
@zimbatm As noted in https://github.com/NixOS/nixpkgs/pull/32285/commits/0520be305a95f70d92b312a7f3a06daf5b51363b these are unusable outside of velox.